glossary.yml

```
@@ -1,4 +1,4 @@
-- slug: 68_95_997_rule
+- Slug: 68_95_997_rule
```

```suggestion
- slug: 68_95_997_rule
```

This must be a lowercase 's'